internal/cmd/local/command_kafka.go

```diff
@@ -0,0 +1,45 @@
+package local
+
+import (
+	"context"
+
+	"github.com/docker/docker/client"
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/internal/pkg/errors"
+)
+
+const (
+	confluentLocalImageName     = "confluentinc/confluent-local:latest"
```

Comment:
We're inside the Confluent Local package, so we don't need to prefix everything with "Confluent Local", but we should probably be clear that this is for Docker.
```suggestion
	dockerImageName     = "confluentinc/confluent-local:latest"
```

Reply:
good point!